src/ProjectOrigin.WalletSystem.IntegrationTests/Jwt/JwtTests.cs

```diff
@@ -225,12 +228,78 @@ public async Task JwtVerification_InvalidData()
 
         // Assert
         var rpcException = await testMethod.Should().ThrowAsync<RpcException>();
-        rpcException.WithInnerException<ValidationException>().WithMessage("Issuer key could not be imported as type ”ecdsa”, No supported key formats were found. Check that the input represents the contents of a PEM-encoded key file, not the path to such a file. (Parameter 'input')");
+        rpcException.WithInnerException<ValidationException>().WithMessage("Issuer key could not be imported as type ”rsa”, No supported key formats were found. Check that the input represents the contents of a PEM-encoded key file, not the path to such a file. (Parameter 'input')");
+    }
+
+    [Fact]
+    public async Task JwtVerification_Authority_Valid()
+    {
+        // Arrange
+        var wireMockServer = WireMockServer.Start();
+        var jwtTokenIssuerFixture = new JwtTokenIssuerFixture()
+        {
+            Issuer = wireMockServer.Urls[0]
+        };
+
+        wireMockServer.Given(Request.Create().WithPath("/.well-known/openid-configuration").UsingGet())
+            .RespondWith(Response.Create().WithBody(jwtTokenIssuerFixture.GetJsonOpenIdConfiguration()));
+        wireMockServer.Given(Request.Create().WithPath("/keys").UsingGet())
+            .RespondWith(Response.Create().WithBody(jwtTokenIssuerFixture.GetJsonKeys()));
+
+        var jwtConfiguration = new Dictionary<string, string?>()
+        {
+            {"Jwt:Authority", jwtTokenIssuerFixture.Issuer},
```

Comment:
Wait a minute, it IS the ISSUER! I've been bamboozled

Reply:
When using authority, it is the same.